test/youdo/db_test.clj

```diff
@@ -0,0 +1,23 @@
+(ns youdo.db-test
+  (:require [youdo.db :as db]
+            [clojure.test :refer [is deftest]]
+            [clojure.java.io :as io]))
+
+(def test-file-1 (io/resource "test/tasks-list.youdo"))
+(def test-action "Laundry")
+
+(def random-file-path (str "test/" (.toString (java.util.UUID/randomUUID)) ".youdo"))
```

Comment:
That random name will be generated for _all the tests_ if you put it here. It would be better to create it _for each test_ instead